### PR TITLE
fix windows build

### PR DESCRIPTION
**Pull request by chrisnojima**
delegate all npm scripts to node helper. added post install step to fixup symlinks in windows

@zanderz can you test

### CODE REVIEW

**Comment by chrisnojima** [comment]
you have to do an npm install when you download this to fix up the symlinks

File: desktop/npm-helper.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+// Helper for cross platform npm run script commands
```
Comment (chrisnojima) [inline desktop/npm-helper.js]: mainly to deal with windows which doesn't have a bash like shell

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by zanderz** [comment]
lgtm, thanks Chris! :thumbsup:

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**